bytecode/concrete.py

```diff
@@ -292,7 +291,7 @@ def to_code(self):
                               self.kwonlyargcount,
                               nlocals,
                               stacksize,
-                              self.flags,
+                              self.flags.to_int(self),
```

Comment:
Python has standard methods to cast to int: int(obj) calls obj.__int__() or obj.__index__().